virtualizarr/tests/test_backend.py

```diff
@@ -13,9 +12,17 @@
 from virtualizarr.backend import FileType
 from virtualizarr.manifests import ManifestArray
 from virtualizarr.readers.kerchunk import _automatically_determine_filetype
-from virtualizarr.tests import has_astropy, has_tifffile, network, requires_s3fs
+from virtualizarr.tests import (
```

Comment:
This is a great addition! Thanks for implementing it.

Reply:
Another good idea just stolen from xarray 😅

Reply:
The source of all good code